datadog_checks_base/CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # CHANGELOG - datadog_checks_base
 
+## 5.0.0 / 2019-01-04
+
+* [Added] Add kube_controller_manager integration. See [#2845](https://github.com/DataDog/integrations-core/pull/2845).
+* [Added] Add kube_leader mixin to monitor leader elections. See [#2796](https://github.com/DataDog/integrations-core/pull/2796).
+* [Added] Add kube_leader mixin to monitor leader elections. See [#2796](https://github.com/DataDog/integrations-core/pull/2796).
```

Comment:
Twice?

Reply:
Good catch, had to rerun the command a couple times as some PRs were missing Changelog labels, perhaps thats why this came in twice? I removed manually for now.